src/qml/SimPanel.qml

```diff
@@ -50,13 +50,15 @@ Component {
             simManager.modemPath = manager.modems
         }
 
-        OfonoManager {
-            id: manager
-        }
+        property list<QtObject> _data: [
```

Comment:
because you can't put QtObject into Column, you have to create a "container" for it.